content/organization/events/speakers/index.en.md

```diff
@@ -27,6 +27,47 @@ weight: 53
 
 * Remember the [rules about Prioritization of Underrepresented/Minority Genders](/about/mission/#r-ladies-rules--guidelines).
 
+* In line with R-Ladies’ Mission Statement and with our [Black Lives Matter Statement](https://blog.rladies.org/post/2020-06-06-blm/) 
+to achieve participation and representation in our events of generally excluded groups, taking into account the intersectionality of gender, race, language and geography; for these reasons we request:
+
+  - In chapter events, invite speakers of diverse race, language and geography (i.e., avoid a one year period with all white speakers at the chapter events).
+  - For panel discussions of 3 or more panelists, invite speakers of diverse race, language and geography (i.e., avoid all white panels).
+
+### How to find speakers? 
+- You can use the [R-Ladies Directory](https://rladies.org/directory/) to identify potential speakers that are not: white, native anglophone, or from high income level countries.
+- You can reach other communities to request for speakers information (see lists below).
```

Comment:
```suggestion
- You can reach out to other communities to request for speakers information (see lists below).
```

Furthermore should we add something (as discussed elsewhere) about not putting too much work on minorities' plates?

Reply:
E.g. a sentence about first looking whether they have a directory.

Reply:
I agree, looking for a directory first would be good and I would also drop the word "request" (because, to me, that sounds a bit like an entitlement to be given suggestions).

```suggestion
- You can check similar directories and reach out to other communities (see lists below).
```